build failed

49s
Run npx --no-install next build
⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

  ▲ Next.js 14.2.13

   Creating an optimized production build ...
Generated 9 documents in .contentlayer
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
<w> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Parsing of /home/runner/work/data-blog/data-blog/embuild-analyses/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js for build dependencies failed at 'import(filePathJoin(generatedPkgPath, 'generated', 'index.mjs'))'.
<w> Build dependencies behind this expression are ignored and might cause incorrect cache invalidation.
 ✓ Compiled successfully
   Linting and checking validity of types ...
Failed to compile.

./src/components/analyses/vastgoed-verkopen/VastgoedDashboard.tsx:131:11
Type error: Conversion of type '{ property_types: { code: string; nl: string; }[]; regions: { code: string; name: string; }[]; provinces: { code: string; name: string; }[]; }' to type '{ municipalities: GeoEntity[]; }' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'municipalities' is missing in type '{ property_types: { code: string; nl: string; }[]; regions: { code: string; name: string; }[]; provinces: { code: string; name: string; }[]; }' but required in type '{ municipalities: GeoEntity[]; }'.

  129 |
  130 | function useMunicipalityOptions(): GeoEntity[] {
> 131 |   return (lookups as { municipalities: GeoEntity[] }).municipalities ?? []
      |           ^
  132 | }
  133 |
  134 | // Geo filter inline component
Error: Process completed with exit code 1.

--- a/embuild-analyses/src/components/analyses/vastgoed-verkopen/VastgoedDashboard.tsx
+++ b/embuild-analyses/src/components/analyses/vastgoed-verkopen/VastgoedDashboard.tsx
@@ -221,27 +221,31 @@ function GeoFilterInline({
                 </CommandItem>
               ))}
             </CommandGroup>
-            <CommandSeparator />
-            <CommandGroup heading="Gemeente">
-              {municipalities.slice(0, 100).map((m) => (
-                <CommandItem
-                  key={m.code}
-                  value={m.name}
-                  onSelect={() => {
-                    onSelect("municipality", m.code)
-                    setOpen(false)
-                  }}
-                >
-                  <Check className={cn("mr-2 h-4 w-4", selectedLevel === "municipality" && selectedNis === m.code ? "opacity-100" : "opacity-0")} />
-                  {toTitleCase(m.name)}
-                </CommandItem>
-              ))}
-              {municipalities.length > 100 && (
-                <CommandItem disabled>
-                  <span className="text-xs text-muted-foreground">... en {municipalities.length - 100} meer</span>
-                </CommandItem>
-              )}
-            </CommandGroup>
+            {municipalities.length > 0 && (
+              <>
+                <CommandSeparator />
+                <CommandGroup heading="Gemeente">
+                  {municipalities.slice(0, 100).map((m) => (
+                    <CommandItem
+                      key={m.code}
+                      value={m.name}
+                      onSelect={() => {
+                        onSelect("municipality", m.code)
+                        setOpen(false)
+                      }}
+                    >
+                      <Check className={cn("mr-2 h-4 w-4", selectedLevel === "municipality" && selectedNis === m.code ? "opacity-100" : "opacity-0")} />
+                      {toTitleCase(m.name)}
+                    </CommandItem>
+                  ))}
+                  {municipalities.length > 100 && (
+                    <CommandItem disabled>
+                      <span className="text-xs text-muted-foreground">... en {municipalities.length - 100} meer</span>
+                    </CommandItem>
+                  )}
+                </CommandGroup>
+              </>
+            )}
           </CommandList>
         </Command>
       </PopoverContent>

